main.js

```diff
@@ -0,0 +1,85 @@
+
+
+console.log("you are here");
+
+
+// create variables needed
+var count;
+var row;
+var container;
+var expAmt;
+var expName;
+var addExpAmt;
+var addUpExp;
+var removeExp = document.querySelector('span');
+var tableEntExp;
+var tableEntName;
+
+
+// this function runs when the button is clicked
+addExpense = function(event){
+  // get the values from the input fields
+
+  addExp = document.createElement('div');
+  expAmt = document.querySelector('.expAmt').value;
+  expName = document.querySelector('.expName').value;
+  console.log('did I get exp amt and name?');
+
+      // counter
+     addUpExp = document.querySelectorAll('.row').length;
```

Comment:
Rather than adding up the number of __rows__, you want to add up the value entered in, which is `expAmt`